src/main/scala/temple/builder/project/ProjectBuilder.scala

```diff
@@ -53,30 +53,56 @@ object ProjectBuilder {
       case (_, service) => service.lookupMetadata[ServiceAuth].nonEmpty
     }
 
+    // Auth service block only used if usesAuth == true
+    val authServiceBlock: ServiceBlock = ServiceBlock(
+      attributes = Map(
+        "id"       -> Attribute(AttributeType.UUIDType, valueAnnotations = Set(Annotation.Unique)),
+        "email"    -> Attribute(AttributeType.StringType(), valueAnnotations = Set(Annotation.Unique)),
+        "password" -> Attribute(AttributeType.StringType()),
+      ),
+    )
+    authServiceBlock.setParent(templefile)
+
     val databaseCreationScripts = templefile.services.map {
```

Comment:
Surely something like this would be nicer to read, rather than duplicating all this code

```
val services = templefile.services + (if (usesAuth) ("auth", authServiceBlock) else ())
```

Reply:
Or if `templefile` could expose a way to add additional services?